### ISSUE TITLE
WCAG 4.1.2: Ensure buttons have discernible text (.Header-searchItem)

### ISSUE THREAD

**Issue by Erikapp** [Erikapp/accessibility-test#1]
#### Issue
`Ensure buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Aeroméxico](https://www.aeromexico.com/pt-br)
#### Element path
.Header-searchItem
#### Snippet
`<button class="Header-searchItem"><span class="Icon Icon--search"></span></button>`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element does not have an implicit (wrapped) <label>
      Element does not have an explicit <label>
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 137.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
